lib/facebook_ads.rb

```diff
@@ -1,3 +1,36 @@
 # frozen_string_literal: true
+
+require 'facebook_ads/client'
+require 'facebook_ads/configuration'
+require 'facebook_ads/errors/not_configured_error'
+
+models_dir = File.join(File.dirname(__FILE__), 'facebook_ads', 'models')
+Dir[File.join(models_dir, '*.rb')].each do |file|
+  require file
+end
+
 module FacebookAds
+  class << self
+    def configure
+      yield configuration
+      @client = nil
+    end
+
+    def configuration
+      @configuration ||= Configuration.new
+    end
+
+    def client
+      raise NotConfiguredError unless configuration.configured?
```

Comment:
Maybe move this check into the client and use the whole configuration object as a parameter to Client.new(...).
At the moment this check is effectively duplicated in the initialiser of the Client with slightly different code.